taskmanager/features/bootstrap/FeatureContext.php

```diff
@@ -15,6 +15,9 @@ class FeatureContext implements Context
     /** @var \App\Application\Task\TaskRegistry */
     private $taskRegistry;
 
+    /** @var \App\Application\Report\ReportRegistry */
```

Comment:
we should not put everything under a single context. 
http://behat.org/en/latest/user_guide/context.html 
maybe sth like that `ReportContext` and `TaskContext` ?